wheatley/main.py

```diff
@@ -183,6 +183,13 @@ def main():
         type=str,
         help="The URL of the server to join (defaults to 'https://ringingroom.com')"
     )
+    parser.add_argument(
+        "-n", "--name",
+        default="",
+        type=str,
+        help="The display name in ringing room to ring assigned bells."
```

Comment:
Need to improve this wording

Reply:
I guess we can decide what to say once we are sure of what exactly that arg is going to do.